HorusMessenger.py

```diff
@@ -108,16 +115,16 @@ def process_udp(udp_packet):
 		pass
 
 def udp_rx_thread():
-	global udp_listener_running
+	
 	s = socket.socket(socket.AF_INET,socket.SOCK_DGRAM)
 	s.settimeout(1)
 	s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-	s.bind(('',HORUS_UDP_PORT))
+	s.bind(('',HP.HORUS_UDP_PORT))
 	print("Started UDP Listener Thread.")
 	udp_listener_running = True
-	while udp_listener_running:
+	while udp_listener_running:    #FIXME this while has no bound or pre-predicted exit, sure there's a socket timeout exception but that's still not very good.
```

Comment:
Remove this comment. The design of the program is that it continues forever until you ctrl+c it.

If you want to add a handler for ctrl+c you could, so that it prints an nice error message when exiting. I suggest opening a new issue to track the TODO: here: https://github.com/projecthorus/HorusGroundStation/issues/new